molecule/fail2ban/goss/default.yaml.j2

```diff
@@ -3,6 +3,13 @@
 package:
   fail2ban:
     installed: true
+  # See: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=770171
+  python3-systemd:
+{% if ansible_facts.distribution_release in ['bookworm'] %}
```

Comment:
peut-être plus simple avec

installed: {{ (ansible_facts.distribution_release in ['bookworm']) | ternary('true', 'false') }}

Reply:
oui beaucoup plus digest !